src/js/burials/config/form.js

```diff
@@ -162,7 +162,7 @@ const formConfig = {
                 widgetClassNames: 'usa-input-medium'
               },
               'ui:errorMessages': {
-                pattern: 'File number must be 8 digits'
+                pattern: 'Your VA file number must be between 7 to 9 digits.'
```

Comment:
Are periods common at the end of validation messages? Just asking as it looks like the previous message didn't have one.

Reply:
@peggygannon asked for it, but no, I don't think most of our messages have them.

Reply:
@peggygannon should we be adding periods to the end of validation messages of a certain length (or even all of them)?

Reply:
@cehsu I had originally indicated to add a period, but if previous validation messages don't have them, then I'm OK with keeping that style.

Reply:
I will go look through the ones we have to be sure.

Reply:
@cehsu @jbalboni one more thought... it's be great to audit our validation messages to see if they have/don't have periods.

Reply:
I took a quick look, I didn't see an instance of one with a period.

Reply:
@jbalboni OK, great. Thanks for checking!